libpacaur/pkgs.sh

```diff
@@ -300,7 +305,7 @@ MakePkgs() {
         # build
         Note "i" $"Building ${colorW}${pkgsdeps[$i]}${reset} package(s)..."
 
-        # install then remove binary deps
+        # build then remove binary deps
```

Comment:
_Install_ was correct, since it refers to repo dependencies. Here again to handle `-Sw` correctly. Note I'm using "repo dependencies" and "binary dependencies" interchangeably, it might be a good idea to stick to one of them only (`repo` would have my preference since it is less confusing).

The seemingly complexity of that code is due to split package support. See b0d4860006f6e908b73dd43f7d9034feed18d486 and related  6b2c1ed9726538434cfae0865c637579be430765.